contrib/debian/openzfs-zfsutils.install

```diff
@@ -131,5 +131,4 @@ usr/share/man/man8/zstreamdump.8
 usr/share/man/man4/spl.4
 usr/share/man/man4/zfs.4
 usr/share/man/man7/zpool-features.7
-usr/share/man/man7/dracut.zfs.7
```

Comment:
Looks like it is valid man page, what's the reason not to install it?

Reply:
We have a separate package for dracut, i.e. `openzfs-zfs-dracut` . The man page is present and installed from there.

By mistake, this was included in openzfs-zfsutils package last time.